### DISCUSSION TITLE
Fix documentation on assertHasNoErrors and assertHasErrors

### DISCUSSION THREAD

**Discussion by julesgraus** [livewire/livewire#6184] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.6

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Have a look at ```assertHasErrors``` and ```assertHasNoErrors``` in the [sourcecode](https://github.com/livewire/livewire/blob/main/src/Features/SupportValidation/TestsValidation.php).

They both accept 1 parameter called ```$keys```. Yet the documentation over [here](https://github.com/livewire/livewire/blob/main/docs/testing.md) and [here](https://github.com/livewire/livewire/blob/main/docs/validation.md), states that i can receive 2 arguments in some cases.

I will create a PR to fix these docs.

### Additional information

Edit: Here is the PR: https://github.com/livewire/livewire/pull/6185

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@julesgraus FYI for docs fixes, if you're going to submit a PR, don't worry about submitting a discussion too. Saves you the effort of having to do both and less for us to go through 🙂

**Reply by julesgraus** [reply]
Ok. Good to know. Thanks.
